Swift/BRCrypto/BRCryptoSystem.swift

```diff
@@ -1753,19 +1753,22 @@ extension System {
                                                             let timestamp = transaction.timestamp.map { $0.asUnixTimestamp } ?? 0
                                                             let height    = transaction.blockHeight ?? 0
                                                             transaction.transfers.forEach { (transfer: BlockChainDB.Model.Transfer) in
-                                                                cwmAnnounceGetTransferItemGEN(cwm, sid, transaction.hash,
-                                                                                              transfer.source, transfer.target,
-                                                                                              transfer.amountValue,
-                                                                                              transfer.amountCurrency,
-                                                                                              timestamp, height)
+                                                                if (accountAddress == transfer.source ||
```

Comment:
We will assert later if we add a transfer that does not belong to us.  There seems to be a bug in BDB that gives us fee transfers for other users (it was fixed once but crept back in)